uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -847,6 +847,76 @@ public function testFindOverlappingLeaveRequestsForOneOverlappingLeaveRequest()
     $this->assertEquals($leaveRequest1->id, $overlappingRequests[0]->id);
   }
 
+  public function testManagerCanCancelOrRejectUserLeaveRequestEvenIfBalanceChangeIsGreaterThanPeriodEntitlementBalanceChangeWhenAllowOveruseFalse() {
+    $manager = ContactFabricator::fabricate();
+    $staff = ContactFabricator::fabricate();
+    $entitlementFromDate = CRM_Utils_Date::processDate('2016-01-01');
+    $entitlementToDate = CRM_Utils_Date::processDate('2016-12-31');
+    $requestFromDate = new DateTime('2016-06-14');
```

Comment:
Maybe use `$requestDate` here too as one variable since its the same for from_date and to_date ?
Also you can use `CRM_Utils_Date::processDate` for the date too instead of having a date time object and then getting the format.

Reply:
Done.